waiter/config-full.edn

```diff
@@ -830,6 +830,14 @@
             :not-found "Not found"
             :prestashed-tickets-not-available "Prestashed jobsystem tickets not available"}
 
+ ;; Waiter launches a separate thread pool executor to perform blocking reads on incoming request bodies
+ :stream-reader {;; the maximum number of concurrent input stream reads allowed in the executor (default 150)
+                 :concurrency-level 150
+                 ;; the idle time before cached threads from the executor are terminated (default 5)
+                 :keep-alive-mins 5
+                 ;; the number of concurrent requests that are supported by the executor (default 50000)
+                 :queue-limit 50000}
+
```

Comment:
How did you come up with these numbers?

Reply:
These are configurable and we can tweak them during launch if required. They have been set high enough to avoid issues under load we introduce in testing.